CHANGELOG.md

```diff
@@ -4,6 +4,21 @@
 
 ## Important Notes
 
+## Breaking Changes
+
+## Changes since v7.0.0
+
+# V7.0.0
+
+## Release Highlights
+
+- Major internal improvements to provider interfaces
+- Added group authorization support
```

Comment:
Might be worth spelling out the specific providers that gained new AuthZ functionality in this AuthZ focused release?

Gitlab projects & keycloak multiple groups I think looking at the CHANGELOG?

Worth calling out new header injectors setting the stage for user-defined session field -> header mapping that Keycloak Gatekeeper users have been asking for after that project's sunset?

Reply:
Good suggestions, have updated, please take another look